force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -378,6 +447,25 @@ public with sharing class ClassesObjectsMethods {
      * @return The Person object of the older person.
      */
     public static Person getOlderPerson(Person person1, Person person2) {
-        return null; // Replace null with the variable you used to store the result
+       if(person1 == null || person2 == null){
+                return null;
+       }
+
+       Integer p1 = person1.getAge();
+       Integer p2 = person2.getAge();
+
+       if(p1 > p2){
+        return person1;
+         }
+       
```

Comment:
See if you can remove this if statement since it's covered by the else statement